### PR TITLE
Added org directive

### PR DESCRIPTION
**Pull request by Oliwerix**
# AAAAAAAAAAAAAAAAssembler
added org directive to AAAAAAAAAAAAAAssembler

### CODE REVIEW

File: asm.py [hunk 1]
```diff
@@ -7,12 +7,17 @@
...
-    # TODO add .org
     # // predpostavljam praviln encodng fajla, pa itak je sam asci kj hces sploh pisat notr pac halo
     # Tole je naslabsa koda k sm jo kadar kol napisu -OW
     nasm = subprocess.run(["nasm", "-e", filename], stdout=subprocess.PIPE, text=True)
     outs = nasm.stdout
-    with open(f"{''.join(filename.split('.')[:-1])}.out", "wb") as outfile:
+    with open(f"{''.join(filename.split('.')[:-1])}.out", "w+b") as outfile:
+        def writeInstruction(out, noBytes=2):
+            "writes int to current position in file, also checks for possible overwriting"
+            if int.from_bytes(outfile.read(noBytes), "big") != 0:
```
Comment (adimineman) [inline asm.py:17]: Zakaj za brezveze konvertiraš v int :(

**Review by adimineman — APPROVED** [review]
fine

File: asm.py (hunk 1 above)
Comment (Oliwerix) [reply asm.py:17]: To je neumno uprasanje

Comment (adimineman) [reply asm.py:17]: ne pa ne

**Pull request merged**